code/game/objects/items/theft_items.dm

```diff
@@ -49,21 +48,60 @@
 	icon = 'icons/obj/nuke_tools.dmi'
 	icon_state = "core_container_empty"
 	item_state = "metal"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF //Don't want people trying to break it open with acid, then destroying the core.
 	var/obj/item/nuke_core/plutonium/core
+	var/dented = FALSE
+	var/cracked = FALSE
 
 /obj/item/nuke_core_container/Destroy()
 	QDEL_NULL(core)
 	return ..()
 
+/obj/item/nuke_core_container/ex_act(severity)
+	switch(severity)
+		if(EXPLODE_DEVASTATE)
+			if(!cracked)
+				crack_open()
+			return
+		if(EXPLODE_HEAVY)
+			if(!dented)
+				dented = TRUE
+			return
+
+		if(EXPLODE_LIGHT)
+			return
+
+/obj/item/nuke_core_container/examine(mob/user)
+	. = ..()
+	if(!cracked)
+		. += "Fine print on the box reads \"Cybersun Industries secure container, guaranteed thermite proof, assistant proof, and explosive resistant.\""
+	if(dented && !cracked)
+		. += "<span class='notice'>[src] looks dented. Perhaps a bigger explosion may break it.</span>"
+	if(cracked)
+		. += "<span class='warning'>It is broken, and can no longer store objects safely.</span>"
```

Comment:
You can shuffle this around a little in order to only check the `cracked` variable once if you do this:
```suggestion
/obj/item/nuke_core_container/examine(mob/user)
	. = ..()
	if(cracked) // Cracked open
		. += "<span class='warning'>It is broken, and can no longer store objects safely.</span>"
	else if(dented) // Not cracked, but dented
		. += "<span class='notice'>[src] looks dented. Perhaps a bigger explosion may break it.</span>"
	else // Not cracked or dented
		. += "Fine print on the box reads \"Cybersun Industries secure container, guaranteed thermite proof, assistant proof, and explosive resistant.\""
```